Kudu.Core/Functions/FunctionManager.cs

```diff
@@ -476,9 +476,9 @@ public void CreateArchive(ZipArchive zip, bool includeAppSettings = false, bool
         /// <param name="projectName">the {projectName}.csproj</param>
         private static ZipArchiveEntry AddCsprojFile(ZipArchive zip, IEnumerable<ZipArchiveEntry> files, string projectName)
         {
-            const string microsoftAzureWebJobsVersion = "2.1.0-beta1";
-            const string microsoftAzureWebJobsExtensionsHttpVersion = "1.0.0-beta1";
-            const string microsoftNETSdkFunctionsVersion = "1.0.0-alpha6";
+            // TODO different template for functionv2
+            // TODO download git repository when read only
```

Comment:
what does this TODO mean?

Reply:
when user has selected a deployment options (ie, `git push` a csharp function created in VS)
this functionality will return `bin/` and append a `csproj` instead of returning an edible project ( resides under `site\repository`)

Reply:
Got it. That makes sense